express-get-json/index.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+
+app.listen(port, () => {
+  // eslint-disable-next-line no-console
+  console.log(`listening on port ${port}~`);
+});
+
+app.get('/api/grades', function (req, res) {
+  const data = [];
+  for (const key in grades) {
+    data.push(grades[key]);
+  }
+  console.error(data);
```

Comment:
Here we want to send the array of grades back to the client, not just console log it.

```JAVASCRIPT
res.json(data);
```

Reply:
I see! Thank you for the feedback! That makes sense.